config.template.yml

```diff
@@ -269,6 +269,12 @@ session:
   # is restricted to the subdomain of the issuer.
   domain: example.com
 
+  # The remember me duration.
+  # Value of 0 disables remember me.
+  # Value is in seconds, or duration notation. See: https://docs.authelia.com/configuration/session.html#duration-notation
+  # Longer periods are less secure. Currently the default is 1m or 1 month.
+  remember_me_duration: "1m"
```

Comment:
Move this above domain.

Reply:
Done

Reply:
Same goes here, we don't quote unless there's special characters, like for regexes or group definitions.